diagnostic.md

```diff
@@ -21,13 +21,32 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --request GET http://www.example.com/donuts
+
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function getDonuts() {
+    debugger;
+    $.ajax({
+        type: "GET",
+        url: "http://www.example.com/donuts",
+        data: {donuts:list},
+        contentType: "application/json;charset=utf-8",
+        dataType: "json",
+        success: function (result) {
+            debugger;
+            alert(result)
+        },
+        error: function (response) {
+            debugger;
+            alert('error');
+        }
```

Comment:
Using `success` and `error` like this will work. However, we taught you to do this a different way (using `.done` and `.fail`) for a good reason - they use a paradigm called a promise. We're not really doing much with promises yet, but they'll be crucial to building more complex asynchronous processes later in the course.